src/components/modules/VerticalNav/VerticalNav.tsx

```diff
@@ -22,18 +22,35 @@ export const VerticalNav = (props: INavProps) => {
 
 interface IProps extends IReactComponentProps {
 	className?: string;
+	ariaLabel?: string;
+	ariaExpanded?: boolean;
+	ariaControls?: string;
```

Comment:
Would it make sense to add these to `IReactComponentProps` so that they're available for all of our components to accept and use? Something like:

```typescript
export default interface IReactComponentProps<T = HTMLElement> {
    // Rest of interface...
	'aria-checked'?: React.HTMLProps<T>['aria-checked'];
	'aria-label'?: React.HTMLProps<T>['aria-label'];
	'aria-expanded'?: React.HTMLProps<T>['aria-expanded'];
	'aria-controls'?: React.HTMLProps<T>['aria-controls'];
```

It's could be nice to be able to capture those props in a `...restProps` while deconstructing and then spread them all at once on the applicable element. Not a blocker though, just an idea!

Reply:
Done!